testing/conformance/src/vector.rs

```diff
@@ -117,10 +117,7 @@ pub enum RandomnessKind {
 #[derive(Debug, Deserialize_tuple, PartialEq, Eq, Clone)]
 pub struct RandomnessRule {
     pub kind: RandomnessKind,
-    pub dst: i64,
```

Comment:
This is a somewhat unfortunate decrease in test vector granularity that I'm guessing will also break the existing vectors we have? Any clever ideas about how we can prevent this?

Reply:
We could, in theory, preserve the current behavior by mocking in the kernel. But then we wouldn't be testing the randomness function.